command-launcher-plugin/pom.xml

```diff
@@ -14,13 +14,13 @@
         <jenkins.version>2.86-SNAPSHOT</jenkins.version> <!-- TODO pending split -->
         <java.level>8</java.level>
     </properties>
-    <name>Command Launcher Plugin</name>
+    <name>Command Agent Launcher Plugin</name>
     <description>Allows agents to be launched using a specified command.</description>
```

Comment:
NIT "... using a specified system command"

Reply:
I guess it is clear enough from context. Anyway, can always be refined later.